src/chk/chk_engine.c

```diff
@@ -178,25 +210,26 @@ chk_engine_setup_pools(struct chk_instance *ins, bool svc)
 			chk_bk_update_pool(pool_cbk, cpr->cpr_uuid);
 		}
 
-		rc = ds_pool_start(cpr->cpr_uuid);
+		uuid_copy(uuid, cpr->cpr_uuid);
```

Comment:
just curious, why do you need this extra uuid_copy?

Reply:
If the pool_start failed, related "cpr" may be destroyed, but we still log the uuid after that, then copy it out firstly.